packages/aws-cdk-lib/aws-lambda/lib/function-base.ts

```diff
@@ -227,6 +227,19 @@ export interface FunctionAttributes {
   readonly architecture?: Architecture;
 }
 
+/**
+ * Parameters to pass into grantInvoke method
+ */
+export interface GrantInvokeProps {
+  /**
+   * Controls whether to grant invoke access to all function versions. Defaults to `false`.
+   *  - When set to `false`, both the function and functions with specific versions can be invoked.
+   *  - When set to `true`, only the function without a specific version (`$Latest`) can be invoked.
```

Comment:
Is this accurate? I would imagine that at least `functionname:$LATEST` would be invokable, and probably even `functionname:3`, if 3 was the latest published version

Reply:
Thanks for catching this, I think you are right👍